sdk/inc/azure/core/az_mqtt5_rpc.h

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ *
+ * @brief Common include for az_mqtt5_rpc server and client.
+ *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
+ */
+
+#ifndef _az_MQTT5_RPC_H
+#define _az_MQTT5_RPC_H
+
+#include <azure/core/az_span.h>
+#include <stdio.h>
+
+#include <azure/core/_az_cfg_prefix.h>
+
+/**
+ * @brief The default timeout in seconds for subscribing/publishing.
+ */
+#define AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS 10
+/**
+ * @brief The default QOS to use for subscribing/publishing.
+ */
+#ifndef AZ_MQTT5_DEFAULT_RPC_QOS
+#define AZ_MQTT5_DEFAULT_RPC_QOS 1
```

Comment:
```suggestion
#define MQTT_QOS_AT_MOST_ONCE 0
#define MQTT_QOS_AT_LEAST_ONCE 1
#define MQTT_QOS_EXACTLY_ONCE 2
#define AZ_MQTT5_DEFAULT_RPC_QOS MQTT_QOS_AT_LEAST_ONCE
```

Reply:
Maybe these MQTT_QOS defines should be in another location, since they are common for other mqtt-related files perhaps?

Reply:
@RLeclair any issues with me adding QOS defines to az_mqtt5? (or are these already defined somewhere?)